.github/workflows/commit_lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Commit lint
```

Comment:
Optional
We could reduce maintenance overhead by remove this script... however it does "populate" the changelog.md. Which in that case we could look at removing the issue number requirement